packages/design-system/src/components/Tooltip/Tooltip.jsx

```diff
@@ -198,9 +199,10 @@ export class Tooltip extends React.Component {
       : {
           onBlur: () => this.handleBlur(),
         };
+    const bodyElement = document.querySelector('body');
 
     const tooltipContent = () => (
-      <span
+      <div
```

Comment:
I'm not sure if this was actually necessary, but it seems if the purpose of the PR is to prevent interactive content passed in from {title} from being within a span, this also should not be a span.

Reply:
I feel like a `<div>` is a better choice here anyways as it's a container that doesn't need to be used inline

Reply:
Agreed, `<div>` is better here. I was the one that originally suggested using a `<span>` so that it wouldn't cause a break in inline content, but if we're using a portal then that solves that issue.